### PR TITLE
DO NOT MERGE: Release java-algorand-sdk 1.9.0

### PR DESCRIPTION
**Pull request by bricerisingalgorand**
# 1.9.0
- Support AVM 1.0
  - Update TEAL langspec to v5
  - Add function to get app address
  - Regenerate REST API types
  - Support new cucumber tests
  - Enable new cucumber tests
- Allow non-base64 metadata hash and force 32 bytes
- mark contract templates as deprecated
- Added pretty print option to Object Mapper instance
- Delete generator since it has it's own repo now

### CODE REVIEW

**Review by jasonpaulos — APPROVED** [review]
Pending tests, looks good!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+- Support AVM 1.0
+  - Update TEAL langspec to v5
+  - Add function to get app address
+  - Regenerate REST API types
+  - Support new cucumber tests
+  - Enable new cucumber tests
+- Allow non-base64 metadata hash and force 32 bytes
+- mark contract templates as @Deprecated
+- Added pretty print option to Object Mapper instance
+- Delete generator since it has it's own repo now
```
Comment (winder) [inline CHANGELOG.md:11]: I don't think this needs to be called out in the changelog, it doesn't effect the release

Comment (bricerisingalgorand) [reply CHANGELOG.md]: removed 👍

**Review by winder — APPROVED** [review]
LGTM. Please make sure to merge master back into develop once you're finished to make sure the branches are all synchronized.

**Pull request merged**